test/test_torch.py

```diff
@@ -2856,6 +2856,11 @@ def test_use_deterministic_algorithms_beta_warning(self, device):
             with self.maybeWarnsRegex(UserWarning, "torch.use_deterministic_algorithms is in beta"):
                 torch.use_deterministic_algorithms(True)
 
+    def test_tensor_from_list_of_ndarray_warning(self, device):
+        warning_msg = r"Creating a Tensor from a list of numpy.ndarray's is extremely slow"
+        with self.maybeWarnsRegex(UserWarning, warning_msg):
```

Comment:
If @mattip's warning update ever lands (I'm still fighting with our internal systems) this will be a great opportunity to use it!